x-pack/plugins/security_solution/public/common/components/exceptions/add_exception_modal/index.test.tsx

```diff
@@ -30,8 +30,9 @@ import {
 } from '../../../../../common/detection_engine/schemas/response/rules_schema.mocks';
 import { useRuleAsync } from '../../../../detections/containers/detection_engine/rules/use_rule_async';
 import { AlertData } from '../types';
+import { getMockTheme } from '../../../lib/kibana/kibana_react.mock';
```

Comment:
super nit: I prefer to keep these relative imports roughly organized by distance

Reply:
Do you have a neat way of doing so by pressing a few keys? Would be super interested in that!!!